app/controllers/user_controller.rb

```diff
@@ -245,6 +245,8 @@ def signchangeemail
     @user.email = @signchangeemail.new_email
     @user.save!
 
+    sign_in(@user, remember_me: session[:remember_me])
```

Comment:
Made a change in #6510 which means we can just do:

```suggestion
    sign_in(@user)
```